server/api/submission.js

```diff
@@ -110,6 +114,7 @@ exports.post = async (
       width,
       url,
       user: username,
+      background_color: backgroundColor,
```

Comment:
Nit, would it be possible to keep object keys in alphabetical order when you are adding them? It would be great to have a linter do this, but in the absence of that availability, it improves readability and being able to find/search for specific keys later when there is a semantic order.

Reply:
Ya, I try to catch these because I never got the linter to behave.

Reply:
Yeah, it's a bit inconsistent in how they work. Speaking of which, eslint is deprecating formatting: https://eslint.org/blog/2023/10/deprecating-formatting-rules/, so we might have to tweak our prettier and eslint configs to account for that some time in the future.